RELEASING.md

```diff
@@ -1,4 +1,5 @@
 ## Releasing
+1. If this is a new minor (not patch) release, rename the 'akka-http-x.x-stable' reporting project in [WhiteSource](http://saas.whitesourcesoftware.com/) accordingly
```

Comment:
Use `https://`, at least it redirects to an HTTPS URL for me.

Reply:
Oops!